src/screens/GroupSetting/index.js

```diff
@@ -92,9 +91,9 @@ const GroupSetting = ({navigation, route}) => {
 export default GroupSetting;
 
 const styles = StyleSheet.create({
-  container: {flex: 1, backgroundColor: '#fff'},
+  container: {flex: 1, backgroundColor: COLORS.white},
   subtitleStyle: {
-    color: COLORS.holytosca
+    color: COLORS.anon_primary
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The color values have been updated to use the new `COLORS` object. This change aligns with the PR's objective of standardizing color usage across the application. The new colors being used (`COLORS.white` and `COLORS.anon_primary`) should be verified to ensure they match the intended design.



<!-- This is an auto-generated comment by OSS CodeRabbit -->